# This service contains the definition for a request for benchmark
# to the planning pipeline

# The scene to consider in the benchmark
PlanningScene scene

# The problem to benchmannnnbrk
MotionPlanRequest motion_plan_request

# Number of runs to execute (so that the information can be averaged later)
uint32 average_count

# The file where to store the results
string filename

---

# The first solution that was found, if any was found
MotionPlanDetailedResponse response

# Location where results were saved
string filename

# Error code - encodes the overall reason for failure
MoveItErrorCodes error_code
